app/assets/javascripts/controllers/ansible_credentials/ansible_credentials_form_controller.js

```diff
@@ -58,13 +58,13 @@ ManageIQ.angular.app.controller('ansibleCredentialsFormController', ['$window',
 
   vm.saveClicked = function(angularForm) {
     API.put('/api/authentications/' + credentialId, vm.credentialModel)
-       .then(getBack(sprintf(__("Modification of Credential \"%s\" has been successfully queued."), vm.credentialModel.name), false))
+       .then(getBack.bind(vm, sprintf(__("Modification of Credential \"%s\" has been successfully queued."), vm.credentialModel.name), false))
```

Comment:
This will need to be either `.then(getBack.bind(vm, sprintf(...), false, false))` (to provide all 3 required params), or `.then(function() { getBack(sprintf(...)); })`.

Otherwise, `response` becomes the 3rd param, causing an error icon instead of success on the flash.